### DISCUSSION TITLE
Order of items in a Collection after mapping

### DISCUSSION THREAD

**Discussion by ihimv** [LuckyPennySoftware/AutoMapper#3809] [NA]
Does AutoMapper preserve the order of items in a collection after it has been auto mapped to some other collection? 

Please refer to this SO question
[Does Automapper preserve mapped list item order?](https://stackoverflow.com/questions/16443356/does-automapper-preserve-mapped-list-item-order)

**Comment by lbargaoanu** [comment]
Check [the execution plan](https://docs.automapper.org/en/latest/Understanding-your-mapping.html). It's unlikely that would change. And even if it does, you can always customize that in your own code.
